subsys/tfm/boards/nrf9160/preload.cmake

```
@@ -0,0 +1,10 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2021, Nordic Semiconductor ASA.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+#-------------------------------------------------------------------------------
+
+set(NRF_PATH platform/ext/target/nordic_nrf)
```

To make it completely clear to the user where this file is located (that is outside this repo), I would suggest:
```suggestion
set(TFM_NRF_DIR ${Trusted\ Firmware\ M_SOURCE_DIR}/platform/ext/target/nordic_nrf)
```
Using `DIR` as that is the common name for a path in CMake.
Also suggesting `TFM_NRF_DIR` to not confuse readers with the `NRF_DIR` in this repo.
https://github.com/nrfconnect/sdk-nrf/blob/master/CMakeLists.txt#L8

---

It was only used here, so I removed the variable.